### PR TITLE
Add Draco 1.3.4 

### PR DESCRIPTION
**Pull request by hobu**
Draco is an Apache 2 C++ library from Google for point cloud and mesh compression/decompression. It works on Win64, Linux, and OSX.

### CODE REVIEW

**Comment by hobu** [comment]
For the same reason described in https://github.com/conda-forge/staged-recipes/pull/7267#issuecomment-446844488 the circle build is failing because the compiler is too old.

File: recipes/draco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  commands:
+    - draco_encoder -h
+
+about:
+  home: https://github.com/google/draco/
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE.txt
```
Comment (chrisburr) [inline recipes/draco/meta.yaml]: I think `LICENSE` in included in the source tarball. If so, you can delete the `LICENSE.txt` file and use this instead:

```suggestion
  license_file: LICENSE
```

File: recipes/draco/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  description: |
+    A library for compressing and decompressing 3D geometric meshes and point clouds
+  doc_url: https://google.github.io/draco/
+  dev_url: https://github.com/google/draco
+
+extra:
+  recipe-maintainers:
+    - hobu
+    - chambbj
```
Comment (chrisburr) [inline recipes/draco/meta.yaml]: @chambbj Are you happy to be listed as a maintainer of this recipe?

Comment (chambbj) [reply recipes/draco/meta.yaml]: @chrisburr More than happy! Please proceed to merge.

**Pull request merged**